VideoLocker/res/values/strings.xml

```diff
@@ -307,6 +307,10 @@
     <string name="discussion_priviledged_author_attribution">By %s</string>
     <!-- The date and author of a discussion comment, question, or thread, e.g. "3 days ago by Brian STAFF" -->
     <string name="post_attribution">{time} by {author} {author_label}</string>
+    <!-- Label for the visibility of a discussion post only visible to cohort -->
```

Comment:
This can be changed to:

> Label for the visibility of a discussion post **that is** only visible to cohort.

... which is clearer and more grammatically correct. Same change can be applied to the "everyone" string as well below.